go/pkg/errors/errors.go

```diff
@@ -0,0 +1,79 @@
+// Package errors provides the ability to wrap multiple errors while maintaining API compatibility with the standard package.
```

Comment:
what's the added value over standard error wrapping with %w?

for instance, instead of ` errors.Join(ErrNotAbsolute, fmt.Errorf("path %q", p))` you could use standard `fmt.Errorf("path %q: %w", p, ErrNotAbsolute)` and [then use errors.Is](https://pkg.go.dev/errors#Is)

Reply:
I'm wrapping 3+ errors in the uploader implementation and would like to keep it consistent throughout the SDK. There are code blocks in the SDK where only the last error is captured while intermediate ones are swallowed which is not great.

Reply:
Understand. Do you think you could add a TODO to remove errors package once we're migrated to go 1.20